app/models/magic_link_feature_metrics.rb

```diff
@@ -7,6 +7,7 @@ def average_magic_link_requests_upto(
     end_time = Time.zone.now.beginning_of_day
   )
     records = ApplicationForm
+      .apply_1
```

Comment:
I'm probs missing something, but does this have to be scoped to apply_1?

Reply:
I think I was concerned about the two getting muddled up because the magic link tokens are linked to the candidate rather than the application form. So it's a very good question though I don't feel confident about just changing it right now. To open it up to apply again applications I basically need to make sure we have a reliable method to avoid counting sign-ins from the previous application.